### PR TITLE
3.2.x changes

### PR DESCRIPTION
**Pull request by DilanUA**

### CODE REVIEW

File: roles/apim-publisher/templates/carbon-home/repository/conf/deployment.toml.j2 [hunk 1]
```diff
@@ -98,4 +97,4 @@ callback_endpoint = "{{ workflow_callback_endpoint }}"
 token_endpoint = "{{ workflow_token_endpoint }}"
 client_registration_endpoint = "{{ workflow_client_registration_endpoint }}"
 client_registration_username = "$ref{super_admin.username}"
-client_registration_password = "$ref{super_admin.password}"
+client_registration_password = "$ref{super_admin.password}"
```
Comment (VimukthiPerera) [inline roles/apim-publisher/templates/carbon-home/repository/conf/deployment.toml.j2:100]: Missing empty line at the EoF.

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 [hunk 2]
```diff
@@ -236,4 +232,15 @@ subscriptions=["POST_ADD_USER"]
...
+[[event_listener]]
+id = "token_revocation"
+type = "org.wso2.carbon.identity.core.handler.AbstractIdentityHandler"
+name = "org.wso2.is.notification.ApimOauthEventInterceptor"
+order = 1
+[event_listener.properties]
+notification_endpoint = "https://localhost:${mgt.transport.https.port}/internal/data/v1/notify"
+username = "${admin.username}"
+password = "${admin.password}"
+'header.X-WSO2-KEY-MANAGER' = "default"
```
Comment (VimukthiPerera) [inline roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2:246]: Missing an empty line at the EoF.

**Review by VimukthiPerera — CHANGES_REQUESTED** [review]

File: roles/apim-publisher/templates/carbon-home/repository/conf/deployment.toml.j2 (hunk 1 above)
Comment (DilanUA) [reply roles/apim-publisher/templates/carbon-home/repository/conf/deployment.toml.j2:100]: Added EOF new line

File: roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2 (hunk 2 above)
Comment (DilanUA) [reply roles/apim/templates/carbon-home/repository/conf/deployment.toml.j2:246]: Added EOF new line

**Review by VimukthiPerera — APPROVED** [review]

**Pull request merged**